src/menu.test.interactions.ts

```diff
@@ -229,23 +230,8 @@ it('remains open when its sub-Menus are opened via click', async () => {
     ),
   );
 
-  await aTimeout(0); // Wait for Floating UI
-
-  // A programmatic click instead of an actual one because `sendMouse()` (via
-  // `mouse.ts`) turned out to be flaky in CI.
-  //
-  // After a bunch of experimentation and source code digging, the ultimate cause of
```

Comment:
Narrator: "The ultimate cause was `aTimeout(0)`".

Reply:
As a followup, I'll replace uses of `aTimeout(0)` (at least ones for Floating UI) throughout the codebase with `requestIdleCallback()`.